vizro-core/tests/unit/vizro/models/_components/test_ag_grid.py

```diff
@@ -378,3 +380,8 @@ def test_aggrid_build_with_description(self, standard_ag_grid):
         )
 
         assert_component_equal(ag_grid, expected_ag_grid, keys_to_strip={"id"})
+
+
+# TODO PP: Add tests
```

Comment:
This would indeed be nice to do if you have time to do it, which it seems like you do. It should be pretty easy to write I think given that `_get_value_from_trigger` is a pure function and I changed the argument `Action` to `value`.